x/ibc/04-channel/keeper/timeout.go

```diff
@@ -42,10 +44,6 @@ func (k Keeper) TimeoutPacket(
 		return nil, types.ErrChannelCapabilityNotFound
 	}
 
-	if !k.portKeeper.Authenticate(portCapability, packet.GetSourcePort()) {
```

Comment:
we don't authenticate the capability anymore? If so add a comment

Reply:
Is this intended? Sounds like we might want to keep this check...

Reply:
Added comment!

Reply:
> Added comment! 

where?

Reply:
Right above this line